### ISSUE TITLE
`ClassSignature` doesn't expose multiline configuration parameter

### ISSUE THREAD

**Issue by FilippoVigani** [detekt/detekt#9083]
Currently `ClassSignature` [hardcodes](https://github.com/detekt/detekt/blob/main/detekt-rules-ktlint-wrapper/src/main/kotlin/dev/detekt/rules/ktlintwrapper/wrappers/ClassSignature.kt) `ktlint_class_signature_rule_force_multiline_when_parameter_count_greater_or_equal_than` to `unset`, instead of providing a configuration parameter.

```kotlin
override fun overrideEditorConfigProperties(): Map<EditorConfigProperty<*>, String> =
        mapOf(
            INDENT_SIZE_PROPERTY to indentSize.toString(),
            MAX_LINE_LENGTH_PROPERTY to maxLineLength.toString(),
            FORCE_MULTILINE_WHEN_PARAMETER_COUNT_GREATER_OR_EQUAL_THAN_PROPERTY to "unset",
        )
```

## Expected Behavior of the rule
Users should be able to change the value of `FORCE_MULTILINE_WHEN_PARAMETER_COUNT_GREATER_OR_EQUAL_THAN_PROPERTY`, for example if they want to always force parameters on a new line (even for single parameters) within a class.. Additionally, it should follow ktlint defaults based on the `code_style` property, as per the documentation:

<img width="829" height="387" alt="Image" src="https://github.com/user-attachments/assets/7af9f26a-7c22-47aa-90a0-6230cb2befe8" />

Right now, it's always hardcoded as `unset`.

## Context
- [ktlint rule documentation](https://pinterest.github.io/ktlint/1.8.0/rules/standard/#class-signature)
